src/main/kotlin/net/minestom/dependencies/UnresolvedDependencyException.kt

```diff
@@ -5,6 +5,4 @@ import java.lang.Exception
 /**
  * Thrown when a given dependency could not be found inside the given resolvers
  */
-class UnresolvedDependencyException(msg: String, cause: Throwable?): Exception(msg, cause) {
-    constructor(msg: String): this(msg, null)
```

Comment:
Why use two different constructors when you can just set a default value for the second parameter.